flow/types.js

```diff
@@ -60,4 +60,32 @@ declare module 'icarus-backend' {
     status: number,
     data: any
   }
+
+  // either txInput or txOutput
+  declare type Movement = {
```

Comment:
rather than creating a new term, I would prefer to come up with something from the domain, something like `txInputOutput` is IMHO fine - not gramatically correct, but at least clear

Reply:
but IMHO it would be beneficial to have a separate type for TxInput and TxOutput - that way Flow can actually catch if you are mixing them accidentally.

Where a unified type makes perhaps sense is with the `MovementEntry` type which is used to define the whay in which the inputs/outputs are serialized in the final response. You can leverage Union types in the helper to initialize that  MovementEntry: https://flow.org/en/docs/types/unions/